### PR TITLE
chore: add cookie secret env var

### LINKED ISSUE #690: making COOKIE_SECRET an env var
make `COOKIE_SECRET` an env var instead of checking in `proxy.config.json`

https://github.com/finos/git-proxy/blob/fd638483c70bc84f3f9cedb6e10f8d0a18bafacf/proxy.config.json#L3

### PR DESCRIPTION
**Pull request by Aditya-PS-05**
closes #690

### CODE REVIEW

File: src/config/index.js [hunk 1]
```diff
@@ -100,8 +100,8 @@ const getAPIs = () => {
 };
 
 const getCookieSecret = () => {
-  if (_userSettings && _userSettings.cookieSecret) {
-    _cookieSecret = _userSettings.cookieSecret;
+  if (!_cookieSecret) {
+    throw new Error('COOKIE_SECRET is not defined in the environment variables.');
```
Comment (rgmz) [inline src/config/index.js:104]: Wouldn't this never trigger because of the fallback to `'cookie secret'`?

File: src/plugin.js
Comment (rgmz) [inline src/plugin.js:1]: Are there any non-whitespace changes in this file?

File: src/config/index.js (hunk 1 above)
Comment (Aditya-PS-05) [reply src/config/index.js:104]: Yeah sorry about that, I need to correct it.

**Comment by Aditya-PS-05** [comment]
@rgmz 
Please review the pr as soon as possible please.

**Comment by JamieSlome** [comment]
@Aditya-PS-05 - it looks like you have a prettier or linter that doesn't align with the standard of the project? Are you able to revert non-related changes so the PR includes changes only relevant to your suggested improvement? Massively appreciate your PR and effort ❤️

**Pull request closed without merging**